### PR TITLE
Add CLRACINGF4AIRV3 to CLRACINGF4AIR target

### PR DESCRIPTION
**Pull request by bnn1044**
add CLRACINGF4AIR V3 target.

### CODE REVIEW

**Comment by digitalentity** [comment]
@bnn1044 is this ready?

**Comment by bnn1044** [comment]
Yes. Test on the bench.

**Comment by digitalentity** [comment]
@bnn1044 can you please rebase against development branch?

**Comment by bnn1044** [comment]
@digitalentity Rebase done

**Comment by digitalentity** [comment]
@bnn1044 there are numerous conflicts. Please fix.

File: docs/Cli.md [hunk 1]
```diff
@@ -341,6 +341,8 @@ Re-apply any new defaults as desired.
 |  yaw_deadband  | 5 | These are values (in us) by how much RC input can be different before it's considered valid. For transmitters with jitter on outputs, this value can be increased. Defaults are zero, but can be increased up to 10 or so if rc inputs twitch while idle. |
 |  throttle_tilt_comp_str  | 0 | Can be used in ANGLE and HORIZON mode and will automatically boost throttle when banking. Setting is in percentage, 0=disabled. |
 |  flaperon_throw_offset  | 200 | Defines throw range in us for both ailerons that will be passed to servo mixer via input source 14 (`FEATURE FLAPS`) when FLAPERON mode is activated. |
+|  gimbal_mode  | NORMAL | When feature SERVO_TILT is enabled, this can be either NORMAL or MIXTILT |
+>>>>>>> fc9ab94... add CLRACINGF4AIRV3 TARGET
```
Comment (fiam) [inline docs/Cli.md]: There are conflict markers merged. Also, it looks like this touches a lot of places that it shouldn't.

Comment (bnn1044) [reply docs/Cli.md]: Hmm. Look like the rebase to development I messed up.  Will update later. Thanks.

File: lib/main/Middlewares/ST/STM32_USB_Device_Library/Class/CDC/Src/usbd_cdc.c [hunk 2]
```diff
@@ -0,0 +1,929 @@
+/**
```
Comment (digitalentity) [inline lib/main/Middlewares/ST/STM32_USB_Device_Library/Class/CDC/Src/usbd_cdc.c]: Why this file is here?

File: lib/main/Middlewares/ST/STM32_USB_Device_Library/Class/CDC_HID/Src/usbd_cdc_hid.c [hunk 3]
```diff
@@ -0,0 +1,332 @@
+/*
```
Comment (digitalentity) [inline lib/main/Middlewares/ST/STM32_USB_Device_Library/Class/CDC_HID/Src/usbd_cdc_hid.c]: Same applies for this file - it doesn't belog here

**Review by digitalentity — DISMISSED** [review]
Please remove stale ST CDC middleware files

File: lib/main/Middlewares/ST/STM32_USB_Device_Library/Class/CDC_HID/Src/usbd_cdc_hid.c (hunk 3 above)
Comment (bnn1044) [reply lib/main/Middlewares/ST/STM32_USB_Device_Library/Class/CDC_HID/Src/usbd_cdc_hid.c]: OK. I use the development as base.
I don't know why it is there.

**Comment by fiam** [comment]
This seems fixed now.

**Pull request merged**